src/components/Uploadcare/lib/effects.ts

```diff
@@ -20,23 +20,27 @@ const resize = (width: number | null = null, height: number | null = null) => {
     }
 
     return `/resize/${safeWidth}x${safeHeight}/`;
-};
+}
 
-export const crop = (width: number, height: number) => {
+export function crop(width: number, height: number) {
     const safeWidth = Math.min(width, MAX_WIDTH);
     const safeHeight = Math.min(height, MAX_HEIGHT);
 
     return `/crop/${safeWidth}x${safeHeight}/`;
-};
+}
 
-export const format = (format: ImageExtension | VideoExtension | "auto") =>
-    `/format/${format}/`;
+export function progressive(value: boolean = true) {
```

Comment:
Do we really need an argument here?
I would assume that you can just not include the effect if you don't need it, similar to how `grayscale` effect already works.

Reply:
The only use-case I can imagine is to cancel the previously applied `progressive` effect with `progressive(false)`. But it's hard to imagine when we actually may need it. :shrug: